modules/ipi-install-node-requirements.adoc

```diff
@@ -37,6 +37,11 @@ Do not deploy a cluster with only one worker node, because the cluster will depl
 * *Network interfaces:* Each node must have at least one network interface for the routable `baremetal` network. Each node must have one network interface for a `provisioning` network when using the `provisioning` network for deployment. Using the `provisioning` network is the default configuration.
 
 * *Unified extensible firmware interface (UEFI):* Installer-provisioned installation requires UEFI boot on all {product-title} nodes when using IPv6 addressing on the `provisioning` network. In addition, UEFI Device PXE Settings must be set to use the IPv6 protocol on the `provisioning` network NIC, but omitting the `provisioning` network removes this requirement.
++
+[IMPORTANT]
+====
+If you are starting the installation from virtual media, delete all old UEFI boot table entries. The only pre-existing boot table entries should be the generic entries provided by the firmware. Otherwise, the installation might fail. 
```

Comment:
You could update "from virtual media" to be more explicit. I am not 100% sure what this means.

Reply:
You could minimise the last two sentences by combining them, as follows:

> If the boot table includes entries that are not generic entries provided by the firmware, the installation might fail.